### PR TITLE
[terra-clinical-item-view] Update docs to use mdx syntax

### PR DESCRIPTION
**Pull request by pranav300**
### Summary
updated docs to use MDX syntax

### CODE REVIEW

File: packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx [hunk 1]
```diff
@@ -11,4 +13,9 @@ const display5 = <ItemView.Display text="Start Time: 08-05-2016 12:00:00" />;
 const display6 = <ItemView.Display text="End Time: 08-05-2016 16:00:00" />;
 const displays = [display1, display2, display3, display4, display5, display6];
 
-export default () => <ItemView displays={displays} layout="twoColumns" />;
+export default () => (
+  <div>
+    <JSXDoc />
```
Comment (emilyrohrbough) [inline packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx]: Why can't we move these example to this file?

Comment (pranav300) [reply packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx]: I think this documentation is redundant as the same has been used for creating the example.

File: packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/clinicalItemView.1.doc.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Component Features
+* [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+* [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
+* [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
+
+[View component source code](https://github.com/cerner/terra-clinical/tree/master/packages/terra-clinical-item-view)
+
+## Examples
+<ItemViewStandard title="ItemView - Default" />
+<ItemViewTwoColumn title="ItemView - Two Column Layout" description="When including icons as part of an '<ItemView.Display />'', it is recommended to use the iconAlignment='inline' prop for best alignment and wrapping with the text." />
```
Comment (emilyrohrbough) [inline packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/clinicalItemView.1.doc.mdx]:
```suggestion
<ItemViewTwoColumn
  title="ItemView - Two Column Layout"
  description="When including icons in the ItemView.Display, it is recommended to use the iconAlignment='inline' prop for best alignment and wrapping with the text."
/>
```

File: packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx [hunk 3]
```diff
@@ -11,4 +11,8 @@ const display5 = <ItemView.Display text="Start Time: 08-05-2016 12:00:00" />;
 const display6 = <ItemView.Display text="End Time: 08-05-2016 16:00:00" />;
 const displays = [display1, display2, display3, display4, display5, display6];
 
-export default () => <ItemView displays={displays} layout="twoColumns" />;
+export default () => (
+  <div>
```
Comment (emilyrohrbough) [inline packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx]: Do we need this extra div?

Comment (pranav300) [reply packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumn.jsx]: updated here https://github.com/cerner/terra-clinical/pull/630/commits/e0598c8feb11116a5aca962c2d695312dfa5e79c.

File: packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/clinicalItemView.1.doc.mdx (hunk 2 above)
Comment (pranav300) [reply packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/clinicalItemView.1.doc.mdx]: updated here https://github.com/cerner/terra-clinical/pull/630/commits/e0598c8feb11116a5aca962c2d695312dfa5e79c.

**Pull request merged**